src/EpsonProjector.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System;
 
 using Crestron.SimplSharp;
-using Crestron.SimplSharpPro.DeviceSupport;
 using PepperDash.Core;
 using PepperDash.Essentials.Core;
-using PepperDash.Essentials.Core.Bridges;
 using PepperDash.Essentials.Core.Queues;
 using Feedback = PepperDash.Essentials.Core.Feedback;
 using Thread = Crestron.SimplSharpPro.CrestronThread.Thread;
+using PepperDash.Essentials.Core.DeviceTypeInterfaces;
+using System.Collections.Generic;
+using Crestron.SimplSharpPro.DM;
+
 
 namespace EpsonProjectorEpi
 {
-    public class EpsonProjector : EssentialsBridgeableDevice, IRoutingSinkWithSwitching, IHasPowerControlWithFeedback,
-        IWarmingCooling, IOnline, IBasicVideoMuteWithFeedback, ICommunicationMonitor, IHasFeedback
+    public class EpsonProjector : PepperDash.Essentials.Devices.Common.Displays.TwoWayDisplayBase, IHasPowerControlWithFeedback,
+        IWarmingCooling, IOnline, IBasicVideoMuteWithFeedback, ICommunicationMonitor, IHasFeedback, IHasInputs<int>
```

Comment:
The `IHasInputs<T>` interface does **NOT** replace the `IRoutingSinkWithSwitching` interface. This device needs to implement both in order to function correctly in an Essentials environment. 

`IRoutingSinkWithSwitching` -> Provides functionality required for this device to work as a destination for Essentials Routing.
`IHasInputs<T>` -> Provides functionality to describe inputs for use on the UI to allow for manually switching between inputs on the device.